pokete_classes/fightmap.py

```diff
@@ -239,31 +236,29 @@ def get_figure_attack(self, figure, enem):
                                          "What do you want to do?",
                                          state="float"))
         while True:  # Inputloop for general options
-            if _ev.get() == "'1'":
-                _ev.clear()
+            action = get_action()
+            if action in (Action.ACT_1, Action.ACCEPT):
                 attack = self.get_attack(figure.curr.attack_obs)
                 if attack != "":
                     return attack
-            elif _ev.get() == "'2'":
-                _ev.clear()
+            elif action in (Action.ACT_2, Action.RUN):
```

Comment:
```suggestion
            elif action == Action.RUN:
```

Reply:
Do you want to remove the hints telling you to press 1, 2, etc for different menus? You would be forced to use the letter keybinds if this was committed